app/workers/my_anime_list_sync_worker.rb

```diff
@@ -1,10 +1,20 @@
 class MyAnimeListSyncWorker
   include Sidekiq::Worker
 
-  def perform(library_entry_id, method)
+  # data is a hash
+  def perform(data)
```

Comment:
I disagree with this.. It makes it confusing and also the delete is only being done in the service not the worker. My goal was to make it as similar to the library_entry object so I can treat it _almost_ the same.